### ISSUE TITLE
[Feat]: Adding a weekly schedule for users

### ISSUE THREAD

**Issue by aashay28** [gabrysia694/Gym-Junkies#144]
### What feature?

I have an idea Suppose website have a set of exercises that are defined for a particular fitness routine. However, what if a user wants to follow this routine consistently over a longer period of time? To address this concern, a weekly schedule can be added to the routine. This schedule can outline which exercises should be done on which days, how many sets and reps of each exercise should be performed, and any additional instructions or recommendations for the user. By incorporating a weekly schedule, users can better track their progress and ensure they are consistently following the routine. It can also help users stay motivated and accountable by giving them a clear plan to follow each week. Overall, adding a weekly schedule can be a helpful tool for individuals looking to maintain a consistent exercise routine and achieve their fitness goals.

### Code of Conduct

- [ ] I agree to follow this project's Code of Conduct

**Comment by gabrysia694** [comment]
Sounds like a good idea, however it requires a lot of steps to accomplish. How would you start?

**Comment by aashay28** [comment]
> Thank you for your interest,

- To implement the mentioned features, follow these steps:

- Create a "Schedule" route in the website's navigation bar.

- Display four cards representing different fitness goals, such as Weight Loss, Muscle Building, Weight Gain, and Strength Training.

- When a user clicks on a specific card, show a seven-day workout plan using an accordion component. Each day will contain a list of exercises.

- Define each exercise with the recommended sets and repetitions (reps).

> By following these steps, users can easily navigate to the Schedule page, select their desired fitness goal, view a weekly workout plan, and access detailed information about each exercise, including the recommended sets and reps.

**Comment by gabrysia694** [comment]
Would you like to work on any of those steps? @aashay28

**Comment by aashay28** [comment]
Yes, i would like to work on this project @gabrysia694

**Comment by aashay28** [comment]
> @gabrysia694  I have started working on this task.

As i mentioned above two steps are completed.
- Create a "Schedule" route in the website's navigation bar.
- Display four cards representing different fitness goals, such as Weight Loss, Muscle Building, Weight Gain, and Strength Training.

checked the attached screenshot and it looks good to me.
Please let me know if you have any concerns.

![Screenshot 2023-05-25 at 5 36 32 PM](https://github.com/gabrysia694/Gym-Junkies/assets/108337259/ed5414a4-210c-45f7-a30a-77babd86c6a1)

**Comment by Helixjoe** [comment]
@aashay28 Can I also help with the next few steps you want to implement ?

**Comment by gabrysia694** [comment]
> > @gabrysia694  I have started working on this task.
> 
> As i mentioned above two steps are completed.
> 
> * Create a "Schedule" route in the website's navigation bar.
> * Display four cards representing different fitness goals, such as Weight Loss, Muscle Building, Weight Gain, and Strength Training.
> 
> checked the attached screenshot and it looks good to me. Please let me know if you have any concerns.
> 
> ![Screenshot 2023-05-25 at 5 36 32 PM](https://user-images.githubusercontent.com/108337259/240908412-ed5414a4-210c-45f7-a30a-77babd86c6a1.png)

Looks great! The design matches the website's theme really well, and it's easy for users to understand.

**Comment by aashay28** [comment]
@gabrysia694 perfect !
I'm ready to generate a pull request. Would you be able to merge it on high priority ?

**Comment by aashay28** [comment]
@gabrysia694  please assigned me this task and will work on the next steps once the functionality is merged.

**Comment by gabrysia694** [comment]
Just merged it, should I assign you both @aashay28 and @Helixjoe ?

**Comment by aashay28** [comment]
> Just merged it, should I assign you both @aashay28 and @Helixjoe ?

Yes it's fine @gabrysia694

**Comment by gabrysia694** [comment]
Done. If any of you have any questions, I'll be happy to help or suggest something!

**Comment by aashay28** [comment]
Need weekly schedule of the mentioned categories @gabrysia694

**Comment by gabrysia694** [comment]
> Need weekly schedule of the mentioned categories @gabrysia694

Do you mean a planned schedule of exercises? How about we label each added exercise into mentioned categories and then generate the schedule by showing let's say 5 random elements (exercises) from the array?

**Comment by aashay28** [comment]
@gabrysia694 Yes, we could provide category labels to the exercise and display it within a structured 7-day work schedule instead of presenting it randomly.

**Comment by gabrysia694** [comment]
how is it going? do you need any help?

**Comment by aashay28** [comment]
@gabrysia694 was busy in working on my own open source project.

> how is it going? do you need any help?

Need Weekly schedule of the mentioned categories.

**Comment by gabrysia694** [comment]
> @gabrysia694 was busy in working on my own open source project.
> 
> > how is it going? do you need any help?
> 
> Need Weekly schedule of the mentioned categories.

no worries. can't we label exercises from the Guide into those 4 categories and then display randomly the exercises in a weekly schedule? 5 exercises every day
